tests/test_clients.py

```diff
@@ -589,7 +592,9 @@ async def test_crossref_retraction_status():
             title="The Dilemma and Countermeasures of Music Education under the Background of Big Data",
             fields=["title", "doi", "authors", "journal"],
         )
-
-        assert "**RETRACTED ARTICLE** Citation: Jiaye Han." in crossref_details.formatted_citation  # type: ignore[union-attr]
```

Comment:
@geemi725 these `union-attr` errors come from `crossref_client.query` giving a possible `None` value. You can fix by asserting that the query's return is not `None`. Just fyi